src/lib/token_list_gen.ts

```diff
@@ -411,18 +413,21 @@ export const updateArbifiedList = async (
     includeOldDataFields,
     skipValidation,
     ignorePreviousList,
+    prevArbifiedList,
   }: {
     includeOldDataFields: boolean;
     skipValidation: boolean;
     ignorePreviousList: boolean;
+    prevArbifiedList: string | null;
   }
 ) => {
   const arbTokenList = await getTokenListObj(pathOrUrl);
   removeInvalidTokensFromList(arbTokenList);
-  const path =
-    process.env.PWD +
-    '/src/ArbTokenLists/' +
-    listNameToFileName(arbTokenList.name);
+  const path = prevArbifiedList
+    ? prevArbifiedList
+    : process.env.PWD +
```

Comment:
Sorry, I overlooked it on the first pass. Can we use `TOKENLIST_DIR_PATH` instead?
```typescript
const path = prevArbifiedList ?? TOKENLIST_DIR_PATH + listNameToFileName(arbTokenList.name);
```